vite.config.ts

```diff
@@ -5,7 +5,7 @@ import eslint from 'vite-plugin-eslint';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-	base: '/',
+	base: '/DjedAlliance.github.io/',
```

Comment:
Note that the website is using https://djed.one as custom domain.

Will this still work?

Reply:
Reset the base now it will work for the custom domain